### ISSUE TITLE
As data user I want all information on variables in the dataset report

### ISSUE THREAD

**Issue by UteH** [dzhw/metadatamanagement#2383]
I think currently, `variable.generationDetails.description` (not `variable.generationDetails.rule`) and `variable.annotations` are not included in the data set reports - but they contain important information.

We already have the information and it's already part of the variable details page but I think we should also add the information in the pdf.

PDF | details page | URL
-- | -- | --
 ![grafik](https://user-images.githubusercontent.com/18573122/65513341-6417db80-dedb-11e9-9221-d53355e2ac89.png)| ![grafik](https://user-images.githubusercontent.com/18573122/65513620-d8527f00-dedb-11e9-873e-02fc1aa04c20.png)  |  https://metadata.fdz.dzhw.eu/#!/de/variables/var-ssy17-ds1-stufa1agg$
 ![grafik](https://user-images.githubusercontent.com/18573122/65513533-b2c57580-dedb-11e9-9a5c-33d9fb93b681.png) | ![grafik](https://user-images.githubusercontent.com/18573122/65513793-37b08f00-dedc-11e9-974a-9066fa945625.png)    |  https://metadata.fdz.dzhw.eu/#!/de/variables/var-ssy17-ds1-stufa1_a$
  |   |

**Comment by rreitmann** [comment]
Prio 1

**Comment by UteH** [comment]
also `derivedVariables`

**Comment by rbirkelbach** [comment]
@UteH please test https://metadatamanagement-test.cfapps.io/#!/en/data-sets/dat-gra2005-ds1$?page=1&size=10&type=surveys

**Comment by UteH** [comment]
## work in progress
tested for (de/en) 
- [x] `variable.generationDetails.description` (not `variable.generationDetails.rule`) (stu011e_g1o)
- [x] `derivedVariables` (stu011e_g1o)
- [x] `panelIdentifier` (ski01a)
- [x] `variable.annotations` (wgt_t1t2d)

looks good!

**Comment by UteH** [comment]
As discussed with @rbirkelbach this issue's features are implemented.

While testing we discussed if we should add the last missing attributes to the dataset report so we are done for good (for some time). Sorry for the late enlightenment. This would be 
- `variable.generationDetails.rule`, `variable.generationDetails.ruleExpressionLanguage`, 
  - e. g. https://metadata.fdz.dzhw.eu/#!/de/variables/var-gra2005-ds1-astu011k_g5$?page=1&size=10&type=surveys&version=2.0.0
-  `filterDetails.expressionLanguage`, `filterDetails.description.en/de`  
   -  e. g. https://metadata.fdz.dzhw.eu/#!/de/variables/var-gra2005-ds1-bocc40a$?page=1&size=10&type=surveys&version=2.0.0

is there something missing?

I had ignored them before because a) currently we don't use them anymore and b) I think we should change the font to `Courier New` or something ....

I assume, this is a new Issue
